hcctl/Cargo.toml

```diff
@@ -9,6 +9,6 @@ edition = "2018"
 [dependencies]
 anyhow = "1.0.33"
 chrono = "0.4.19"
-clap = { version = "3.0.0-beta.2", default-features = false, features = ["std", "suggestions", "color"] }
+clap = "3.0.0-beta.2"
```

Comment:
Don't do that, just turn on the `derive` feature instead.

Reply:
The default features adds `cargo` which has the `crate_version!()` macro so this can stay as-is